packages/expo-file-system/src/ExponentFileSystem.web.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+class ExponentFileSystem {
```

Comment:
Export an object literal since we don't intend to use this as a instantiable class:

```js
export default {
  name: 'ExponentFileSystem',
  documentDirectory: null,
  ...
  async getInfoAsync() {
    throw new Error(`Invoked an unimplemented platform method`);
  },

  ...
}
```